tasks/section1.yml

```diff
@@ -447,16 +452,18 @@
       - rule_1.1.22
 
 - name: "NOTSCORED | 1.2.1 | PATCH | Ensure package manager repositories are configured"
-  command: /bin/true
+  command: yum check-update
```

Comment:
Will `yum check-update` fail if repos aren't configured? Per the CIS benchmark it asks you to run `yum repolist` and then check the configured repos...not sure how feasible that is to implement though.

Reply:
yum check-update will indeed fail if repo's aren't configured. But when reading section 1.2 of the CIS advice, the goal is actually that there ARE repo's configured. But the system comes with default repo's. Any 'yum install <item> will fail if there is one misconfigured repo, even if that item would come from an existing, correctly configured repo. 
yum repolist will not do any check on the configuration of the repo's, it will simply list the repo's without checking the correct configuration of them. 
And according CIS advice: "The specifics on patch update procedures are left to the organization.". 
Thus, using yum check-update will be sufficient to check if there are repo's configured. Any misconfiguration will be catched with the first 'yum install <item>' or 'yum search <item>'.